modules/SofaBoundaryCondition/SofaBoundaryCondition_test/FixedConstraint_test.cpp

```diff
@@ -156,6 +158,91 @@ struct FixedConstraint_test : public BaseTest
         return true;
 
     }
+
+    bool testTopologicalChanges()
+    {
+        simulation::Simulation* simulation;
+        sofa::simulation::setSimulation(simulation = new sofa::simulation::graph::DAGSimulation());
+        
+        /// Scene creation
+        simulation::Node::SPtr root = simulation->createNewGraph("root");
+        root->setGravity(defaulttype::Vector3(0, 0, 0));
+
+        /// Create euler solver
+        simulation::Node::SPtr node = createEulerSolverNode(root, "test");
+
+        /// Add mechanicalObject
+        sofa::Size nbrDofs = 5;
+        typename MechanicalObject::SPtr dofs = addNew<MechanicalObject>(node);
+        dofs->resize(nbrDofs);
+        
+        /// Add PointSetTopology
+        typename sofa::component::topology::PointSetTopologyContainer::SPtr tCon = sofa::core::objectmodel::New<sofa::component::topology::PointSetTopologyContainer>();
+        typename sofa::component::topology::PointSetTopologyModifier::SPtr tMod = sofa::core::objectmodel::New<sofa::component::topology::PointSetTopologyModifier>();
+        tCon->setNbPoints(nbrDofs);
+        node->addObject(tCon);
+        node->addObject(tMod);
+
+        /// Add Mass
+        createUniformMass<DataTypes>(node, *dofs.get());
+
+        /// Add force
+        typename ForceField::SPtr forceField = addNew<ForceField>(node);
+        
+        // create a force vector
+        Deriv force;
+        for (unsigned i = 0; i < force.size(); i++)
+            force[i] = 10;
```

Comment:
Is it possible to write `Deriv force = { 10 };` ?

Reply:
Error	C2440	'initializing': cannot convert from 'initializer list' to 'sofa::defaulttype::RigidDeriv<3,real>'